roles/api.py

```diff
@@ -162,3 +165,61 @@ def remove_user_from_repo_group(
         repo_group = Group.objects.get(name=group_type.format(repo.slug))
         user.groups.remove(repo_group)
         user.save()
+
+
+def list_users_in_repo(repo, base_group_type=None):
+    """
+    Lists all the users in the repository groups
+    If the group type is specified, the list is limited to that group
+
+    Args:
+        repo (learningresources.models.Repository): repository used to extract
+            the right group to use
+        base_group_type (roles.permissions.BaseGroupTypes): group unicode
```

Comment:
Can you change the type for this to unicode? If we ever use docstring inspection in IDEs or in documentation generation exact types would be useful